src/Services/UnitConversionService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace FurnitureStoreApi\Services;
+
+class UnitConversionService
+{
+    public static function unitConverter(string $unit, int $depth, int $width, int $height): array|string
```

Comment:
Add the unit as a static property and set it from your endpoint `.php` file

Reply:
I think i've done this right now, and hopefully in the right place